src/frontend/src/btc/components/transactions/BtcTransaction.svelte

```diff
@@ -21,11 +21,14 @@
 
 	let label: string;
 	$: label = type === 'send' ? $i18n.send.text.send : $i18n.receive.text.receive;
+
+	let amount: bigint | undefined;
+	$: amount = nonNullish(value) ? (type === 'send' ? value * BigInt(-1) : value) : undefined;
```

Comment:
just to be sure, better use `value.mul(BigNumber.from(-1))` here, you never know with `BigInt`

Reply:
@AntonioVentilii `value` is of type `bigint` and does not provide a function `mul`.

Reply:
ah! then all good! I thought it was `BigNumber`, my mistake, sorry